include/server/Settings.hpp

```diff
@@ -0,0 +1,20 @@
+#include <string>
+#include <filesystem>
+#include <nlohmann/json.hpp>
+
+class Settings
```

Comment:
create `alicia` namespace around the Settings class

Reply:
done